src/domain/Statuses/Status/Status.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+
+import { Props, Variables } from '../../../common'
+import { Text } from '../../Typographies/Text'
+import { StatusDot, StatusDotVariants } from '../StatusDot'
+import { StyledStatus } from './style'
+
+enum Size {
+  Small = 'small',
+  Large = 'large'
+}
+
+interface IStatusProps {
+  /** Component context */
+  componentContext?: string
+  /** What style variant of status dot to display */
+  variant?: StatusDotVariants
```

Comment:
I think this should be required, there isnt really a situation where you don't want to pass in a variant anyway